rfc6962-bis.xml

```diff
@@ -419,7 +419,7 @@ Section 8.2 of <xref target="RFC3447"/>) using a key of at least 2048 bits.
         A log is a single, append-only Merkle Tree of submitted certificate and precertificate entries.
       </t>
       <t>
-        When it receives a valid submission, the log MUST return an SCT that corresponds to the submitted certificate or precertificate. If the log has previously seen this valid submission, it MAY return the same SCT as it returned before (note that if a certificate was previously logged as a precertificate, then the precertificate's SCT would not be appropriate, instead a fresh SCT of type x509_entry should be generated).
+        When it receives a valid submission, the log MUST return an SCT that corresponds to the submitted certificate or precertificate. If the log has previously seen this valid submission, it MAY return the same SCT as it returned before. (Note that if a certificate was previously logged as a precertificate, then the precertificate's SCT of type <spanx style="verb">precert_sct</spanx> would not be appropriate; instead, a fresh SCT of type <spanx style="verb">x509_sct</spanx> should be generated).
```

Comment:
The brackets are consistent with other parts of the doc, ISTM.  If you want to change that, how about doing it throughout the doc in a separate PR?

$ grep " (Note" rfc6962-bis.xml | wc -l
6